hovercontrol/src/hoverkite.rs

```diff
@@ -124,16 +122,21 @@ impl Hoverkite {
                 self.right_last_command_time = Instant::now();
             }
         };
-        match (side, self.left_port.as_mut(), self.right_port.as_mut()) {
-            (Side::Left, Some(port), _) => command.write_to_std(port)?,
-            (Side::Left, None, Some(port)) => SecondaryCommand(command).write_to_std(port)?,
-            (Side::Right, _, Some(port)) => command.write_to_std(port)?,
-            (Side::Right, Some(port), None) => SecondaryCommand(command).write_to_std(port)?,
-            (_, None, None) => error!(
-                "No serial ports available. Can't send command to {:?}: {:?}",
-                side, command
-            ),
-        }
+        let side_command = SideCommand { side, command };
+        let port = match (side, self.left_port.as_mut(), self.right_port.as_mut()) {
+            (Side::Left, Some(port), _) => port,
+            (Side::Left, None, Some(port)) => port,
+            (Side::Right, _, Some(port)) => port,
+            (Side::Right, Some(port), None) => port,
```

Comment:
`panic!("the primary is currently assumed to be Right")` or something? (or just leave it like this until we work out a way to have the boards discover their role)

Reply:
I've left it for now.